.renovaterc.json

```diff
@@ -31,6 +31,87 @@
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["container-service-(?<currentValue>.*?)-fat.jar"],
+      "depNameTemplate": "xnatdev/container-service",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT batch launch plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["batch-launch-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatx/xnatx-batch-launch-plugin",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT dax plugin specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["dax-plugin-genProcData-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "VUIIS/dax",
+      "datasourceTemplate": "github-releases"
+    },
+    {
+      "customType": "regex",
+      "description": "Update OHIF Viewer plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["ohif-viewer-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "icrimaginginformatics/ohif-viewer-xnat-plugin",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT ML plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["ml-plugin-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatx/ml-plugin",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT datasets plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["datasets-plugin-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatx/datasets-plugin",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT image viewer plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["ximgview-plugin(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatdev/xnat-image-viewer-plugin",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT dxm plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["dxm-settings-plugin-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatx/xnatx-dxm-settings-plugin",
+      "datasourceTemplate": "bitbucket-tags",
+      "versioningTemplate": "regex:(?<major>\\d+).(?<minor>\\d+)"
```

Comment:
Is this `versionTemplate` needed? I added it as [the current version is 1.0](https://github.com/UCL-MIRSG/ansible-collection-xnat/blob/6b9ec3b7a8a06e002c7afa83a81e88ddfcc32b90/roles/xnat/defaults/main.yml#L51)

Reply:
Probs better to leave it in just in case the default versioning doesn't pick it up.